magefiles/utils_test.go

```diff
@@ -94,6 +94,14 @@ func (m *QuayClientMock) ChangeRepositoryVisibility(organization, imageRepositor
 	return nil
 }
 
+func (m *QuayClientMock) GetNotifications(organization, repository string) ([]quay.Notification, error) {
+	return nil, nil
+}
+
+func (m *QuayClientMock) CreateNotification(organization, repository string, notification quay.Notification) (*quay.Notification, error) {
+	return nil, nil
+}
+
```

Comment:
what's the need for these functions?

Reply:
Those functions were added after the update of the `image-controller` dependency to be able to implement the [QuayService interface](https://github.com/konflux-ci/image-controller/blob/83b8a0a578bea92c0d945fb3fd1be84e150b8899/pkg/quay/quay.go#L30)